### DISCUSSION TITLE
Notifications Blank After Allowing Spam Folder Submissions

### DISCUSSION THREAD

**Discussion by smockensturm** [solspace/craft-freeform#297] [Questions & Answers] [answered]
Hey I am trying to understand what happens when a submission marked spam (in the spam folder) is released / allowed.

From the docs:

> Email notifications and API integrations will all be suppressed, but queued.

How? I don't see anything in the Craft queue nor the database. How does this work?

We have a case where a legitimate submission is marked as spam and thrown in the spam folder. When released, the notification **does** get sent but it is blank. Empty. There is no submission data.

Where else can I look? Thanks.

**Comment by kjmartens** [comment, marked as answer]
Hi @smockensturm,

Which version of Freeform are you using? I have tested this again now, and it seems to work fine for me. What does your email notification template look like? Perhaps that is part of the issue.

As for how it handles API integrations, you'll see there's a `freeform_integrations_queue` database table. :)

**Comment by smockensturm** [comment]
`Which version of Freeform are you using?`

3.13.0

`What does your email notification template look like? `

Default. Out of the box.

Does the deferred email notification get put into the Craft queueu?

**Reply by kjmartens** [reply]
Oh, you're using the index page's new mass "allow" feature. I can duplicate this issue not working in my testing as well. We'll get a fix for this soon. In the meantime, you can click the "Allow" button inside the individual detail page, which seems to work correctly.

As for the deferred email notification and API integrations, it does not use the Craft Queue feature. I can understand how using that word in the description might have seemed to indicate what happens.
